.eslintrc.js

```diff
@@ -24,12 +19,51 @@ module.exports = {
                 aspects: ['noHref'],
             },
         ],
-        'operator-linebreak': ['error', 'after'],
+        'jsx-a11y/label-has-associated-control': [
+            2,
+            {
+                depth: 5,
+            },
+        ],
         'no-only-tests/no-only-tests': 'error',
-        'import/no-extraneous-dependencies': ['error', { devDependencies: true }],
+        'no-process-env': 'error', // Use getEnv instead
+        'no-restricted-properties': [
+            2,
+            {
+                object: 'window',
+                property: 'config',
+                message:
+                    'Please pass configuration variables to the MFE via props or React context instead of using global variables',
+            },
+        ],
+        'no-underscore-dangle': ['error', { allow: ['__typename'] }],
+        'react/function-component-definition': [
+            'error',
+            {
+                namedComponents: ['function-declaration', 'arrow-function'],
+                unnamedComponents: ['arrow-function'],
+            },
+        ],
+        'react/jsx-indent-props': [2, 4],
+        'react/jsx-indent': [2, 4],
+        'react/jsx-props-no-spreading': 0,
+        'react/jsx-no-bind': 0,
+        'react/jsx-wrap-multilines': ['error', { declaration: false, assignment: false }],
     },
-    plugins: [
-        'compat',
-        'no-only-tests',
+    overrides: [
+        {
+            files: ['**/*.test.jsx', '**/*.test.js'],
+            rules: {
+                'compat/compat': 0,
+                'react/display-name': 0,
+            },
+        },
     ],
+    plugins: ['compat', 'no-only-tests'],
+    settings: {
```

Comment:
Can we not add '"browserslist": [
    "last 2 versions",
    "not ie <= 11",
    "ios_saf >= 12",
    "not op_mini all",
    "not dead"
]',

here instead of writing this in every package.json where this lib is used? Just a question for my understanding!

Reply:
It's a good idea to do that but the docs say it should be in the package.json file.  https://www.npmjs.com/package/eslint-plugin-compat